deno.jsonc

```diff
@@ -1,6 +1,6 @@
 {
   "tasks": {
-    "dev": "deno run --watch src/main.ts",
+    "dev": "deno run --allow-read --allow-write --allow-env --allow-run src/main.ts",
```

Comment:
issue(blocking): this will allow all reads/writes/envs/runs and kills the purpose of deno security feature
Instead, we should follow the [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege).
We will need to derive what specific operations to allow when we get the rest of this PR sorted out, don't worry about it right now. We will keep this message until the very end.